config/initializers/assets.rb

```diff
@@ -4,4 +4,5 @@
   miq_novnc.js
   spice-html5.js spiceHTML5/spicearraybuffer.js
   codemirror/modes/*.js codemirror/themes/*.css
+  container_topology.css
```

Comment:
Used to be needed but it got fixed in container topology, @pilhuhn can you please update your code so that it's not needed? (Though I think you just need to remove it from here, it's included from `application.css` now.)